network-api/networkapi/wagtailpages/models.py

```diff
@@ -782,7 +782,7 @@ class HomepageFeaturedNews(WagtailOrderable, models.Model):
         'wagtailpages.Homepage',
         related_name='featured_news',
     )
-    news = models.ForeignKey('news.News', related_name='+')
+    news = models.ForeignKey('news.News', on_delete=models.CASCADE, related_name='+')
```

Comment:
does this require a migration?

Reply:
nope, I believe this was the default behaviour, they've just made it mandatory to specify now.